x-pack/plugins/apm/public/components/app/mobile/service_overview/stats/index.tsx

```diff
@@ -74,9 +74,9 @@ export function MobileStats({
         defaultMessage: 'Crash Rate',
       }),
       icon: getIcon('bug'),
-      value: data?.crashCount?.value ?? NaN,
+      value: NaN,
```

Comment:
Why `NaN`?

Reply:
Why not make it optional if it can be `undefined`?
```suggestion
```

Reply:
The value expects to be number | string. I will update it to 'N/A'